ledger/test-common/src/main/daml/simple_package_list/Simple.daml

```diff
@@ -0,0 +1,30 @@
+module Simple where
+data SimpleVariant = SV Party
+template SimpleTemplate
+   with
+     owner : Party
+     observer : Party
+     contractData : [Party]
+   where
+     signatory owner
+     observer  observer
+     key owner : Party
+     maintainer key
+     choice Consume : ()
```

Comment:
Isn’t this just `Archive`?

Reply:
It is but to use that I would have to add logic that apart from checking the package id the type is in, it would have to distinguish between modules. The `Archive` choice is in `DA.Internal.Template` while `Consume` and others are in `Simple`. Please let me leave it as is.

Reply:
Can we just set the type constructor id to `None` in `ValueRecord` to avoid having to figure out the package id? That said, not opposed to keeping it if it makes things significantly easier but then let’s please add a comment documenting why we do this.

Reply:
Yup, worked, done. Thanks